### DISCUSSION TITLE
Pre-release for NC25 published

### DISCUSSION THREAD

**Discussion by christianlupus** [nextcloud/cookbook#1315] [Announcements]
Hello to everyone,

I just want to give you a short heads-up: I pushed out version 0.10.0rc1 just now. This is the first version compatible with NC25 of the cookbook.

The 0.10.0 can be seen as a hot-fix to support the NC25 version of the server. Thus, no new functionality was added with this version in comparison to 0.9.17.

The changes were only in the frontend to make it compatible with the most recent updates in the frontend of the server UI. The backend was not touched by this. I hope, I did not miss any significant glitch with the frontend as this was hammered together in a rather quick way.

If you find anything not working as expected, please report back both now (before the final 0.10.0 hits public) for a quick fix. Later it will eventually take a bit more time to be implemented.

I plan to push out the 0.10.0 by the weekend if no big issues are reported back. That way, the updaters to NC25 will be happy hopefully.

Christian

**Comment by MarcelRobitaille** [comment]
I had a look through the diff of #1314. I don't understand all the changes, but it mostly looks good to me. I did notice one small thing and left a comment. I will update my dev environment to NC25 and test the new version.
